source/common/config/utility.h

```diff
@@ -224,6 +224,17 @@ class Utility {
     return {ALL_SUBSCRIPTION_STATS(POOL_COUNTER(scope), POOL_GAUGE(scope))};
   }
 
+  /**
+   * Generate a SubscriptionStats object from stats scope.
+   * @param scope for stats.
+   * @return SubscriptionStats for scope.
+   */
+  static ControlPlaneStats generateControlPlaneStats(Stats::Scope& scope) {
```

Comment:
Where is this used?

Reply:
Oh yeah, looks like it got moved to GrpcStream in my earlier GrpcStream PR, I think? Removed, and seems not to be missed.